### PR TITLE
Revert incorrectly changed files in  #30778

### PR DESCRIPTION
**Pull request by N6REJ**
Pull Request for Issue # .

### Summary of Changes
Reverts incorrectly changed files from  #30778

### Testing Instructions
test per #30778

### CODE REVIEW

File: administrator/manifests/packages/pkg_weblinks.xml [hunk 1]
```diff
@@ -1,30 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (wilsonge) [inline administrator/manifests/packages/pkg_weblinks.xml]: This shouldn't be deleted

Comment (N6REJ) [reply administrator/manifests/packages/pkg_weblinks.xml]: yeah, why is it.. I didn't do that

Comment (N6REJ) [reply administrator/manifests/packages/pkg_weblinks.xml]: @wilsonge i had to "add" it to git to get it to push

**Comment by zero-24** [comment]
Mergeing that whitespace only fix in the readme now by review thanks @N6REJ

**Comment by brianteeman** [comment]
the title of this pr and commit has no relevance to the changes

**Comment by N6REJ** [comment]
go away @brianteeman you clearly can't read logs

**Comment by zero-24** [comment]
Please do not attack each other personally here. This PR here fixed an whitespace issue and i might should have choosen an better commit message, right. When there are personal differences please clear them offline and not at the tracker. Thanks!

**Comment by N6REJ** [comment]
@zero-24 please check https://github.com/joomla/joomla-cms/pull/31296, its related

**Review by N6REJ — COMMENTED** [review]
@wilsonge apparently I had to add more files.. I got everything right this time in https://github.com/joomla/joomla-cms/pull/31296

**Comment by zero-24** [comment]
> @zero-24 please check #31296, its related

Im not sure where that is realted? That an empty PR with 0 files changed? Maybe a commit was missing?

**Comment by brianteeman** [comment]
@N6REJ before attacking you need to look here on github to see what your PR are. Your local branches are clearly messed up.

**Comment by brianteeman** [comment]
@zero-24 nothing personal. Was just pointing out that the commit message has no relevance to a whitespace change. The git commit log has value and should have relevant messages. However it is clear from @N6REJ comments and subsequent PR that he was expecting this PR to do more than a simple white space change

**Comment by N6REJ** [comment]
@zero-24 ok i'm totally confused... when checking my "git log" for things that changed from todays pull from staging it blamed #30778 for 3 files being deleted and 1 file being added.  When I told git to revert that it happily did so but I didn't realize it wouldn't "add" those files back to git which is why they didn't show up the first time.
Now when I check the log it shows they are infact back now.
image

**Comment by brianteeman** [comment]
It really is much easier if you always create branches from joomla-cms and not from a local fork and that you always create a new branch for every pr

**Comment by N6REJ** [comment]
> It really is much easier if you always create branches from joomla-cms and not from a local fork and that you always create a new branch for every pr

thats my general practice..

1. make sure current local branch is staging
2. pull from joomla staging
3. push to n6rej/staging
4. create new branch
5. make changes & commit > push to new branch.
6. create pr
7. switch back to staging

Something wrong with that method?

for updating/syncing the process is almost identical
1. make sure I'm on staging
2. pull from joomla->staging
3. push to n6rej/staging.
4. switch to my local pr branch
5. pull from joomla->staging
6. push to my remote pr branch
7. switch back to staging

**Comment by brianteeman** [comment]
Are you doing this at the command line or with some software. You seem to be working with multiple staging ??
Highly recommend using smartgit. Recording below

git

**Comment by N6REJ** [comment]
I use git in phpstorm mostly.

**Pull request merged**